packages/loader/src/GLTFLoader.ts

```diff
@@ -19,14 +19,18 @@ export class GLTFLoader extends Loader<GLTFResource> {
    * @remarks If use loader after releasing, we should release again.
    */
   static release(): void {
-    MeshoptDecoder.release();
+    getMeshoptDecoder().then((meshoptDecoder) => {
```

Comment:
`getMeshoptDecoder()`：would lead to useless initialization? If meshOPT initialization is never triggered